plugins/ezid/logic.py

```diff
@@ -110,8 +110,9 @@ def send_create_request(data, shoulder, username, password, endpoint_url):
         print("%d %s\n" % (ezid_error.code, ezid_error.msg))
         if ezid_error.fp is not None:
             response = ezid_error.fp.read()
-            if not response.endswith("\n"):
-                response += "\n"
+           # FIXME: this code throws errors, endswith doesn't work with response
+           # if not response.endswith("\n"):
+           #     response += "\n"
```

Comment:
did you mean to leave this commented out?

Reply:
Yes, the code throws an error which makes troubleshooting difficult, it's easier with it commented out. I made a card for working the FIXME, It's PUBD-190